skydive-grafana/kube/grafana-skydive.json

```diff
@@ -0,0 +1,155 @@
+{
+   "kind":"List",
```

Comment:
Sure @etsauer, I can convert this to YAML. Is this the format we have selected by default? I initially started writing everything in YAML, but then moved to JSON as there are not tools (there is a custom Ansible module, but not a core module) to operate YAML files with array entries, or at least I'm not aware about them, and in the other hand 'jq' is a powerful tool to operate a JSON file.